<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtSa_Dialog_FreqSetup</class>
 <widget class="QWidget" name="RtSa_Dialog_FreqSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
#line
{
	background-color:rgba(46,46,46,1);
}

#widget&gt;QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}

#pb_Uplink:checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
}
#pb_Uplink:!checked
{
padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
}

#pb_Downlink:checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
	background-position:right;
}

#pb_Downlink:!checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
	background-position:right;
}

#widget
{
	background-color:rgba(211,211,211,0.9);
}
#widget&gt;QLabel
{
font: 75 13px &quot;Arial&quot;;
}
RtSa_LineEdit
{
	background-color:rgba(255,255,255,0.6);
	color:rgba(162, 115, 0,1);
	padding:8px 8px 7px 10px ;
	font: 75 12px &quot;Arial&quot;;
	font-weight:bold;
}
RtSa_LineEdit:focus
{
  outline: none;
  border:2px solid rgba(162,115,0,1);
}
*/
/*******************indoor**********************/

#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] #widget
{
	background-color:rgba(211,211,211,0.9);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] #line
{
	background-color:rgba(46,46,46,1);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] #widget&gt;QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] #widget&gt;QPushButton:checked
{
	color:rgba(87, 87, 87,1);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] #pb_Uplink:checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
}
#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] #pb_Uplink:!checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] #pb_Downlink:checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
	background-position:right;
}

#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] #pb_Downlink:!checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
	background-position:right;
}

#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] #widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] RtSa_LineEdit,
#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] RtSa_LineEdit:disable
{
	background:rgba(255,255,255,1);
	color:rgba(162, 115, 0,1);
	padding:8px 6px 7px 6px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_FreqSetup[status  =  &quot;indoor&quot;] RtSa_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
	padding:6px 6px 5px 8px ;
}
/*******************outdoor**********************/

#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] #widget
{
	background-color:rgba(211,211,211,0.9);
}
#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] #line
{
	background-color:rgba(46,46,46,1);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] #widget&gt;QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] #widget&gt;QPushButton:checked
{
	color:rgba(87, 87, 87,1);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] #pb_Uplink:checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
}
#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] #pb_Uplink:!checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] #pb_Downlink:checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
	background-position:right;
}

#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] #pb_Downlink:!checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
	background-position:right;
}

#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] #widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] RtSa_LineEdit,
#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] RtSa_LineEdit:disable
{
	background:rgba(255,255,255,1);
	color:rgba(162, 115, 0,1);
	padding:8px 6px 7px 6px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_FreqSetup[status  =  &quot;outdoor&quot;] RtSa_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
	padding:6px 6px 5px 8px ;
}
/*******************night**********************/
#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] #widget
{
	/*background-color: rgba(251, 219, 167,0.9);*/
   background-color:rgba(208,182,139,0.9);
}
#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] #line
{
	background-color:rgba(46,46,46,1);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] #widget&gt;QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] #widget&gt;QPushButton:checked
{
	color:rgba(87, 87, 87,1);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] #pb_Uplink:checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
}
#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] #pb_Uplink:!checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] #pb_Downlink:checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
	background-position:right;
}

#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] #pb_Downlink:!checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
	background-position:right;
}

#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] #widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}

#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] 
RtSa_LineEdit
{
	background: rgba(251,220,167,0.60);
   color:rgba(159,98,0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] 
RtSa_LineEdit:disable
{
color:rgba(133,133,133,0.6);
}
#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;] RtSa_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
}</string>
  </property>
  <property name="status" stdset="0">
   <string>night</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>254</width>
     <height>648</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>117</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start
Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>67</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Center
Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>482</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel
Number</string>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="LedChStd">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>432</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>CDMA-Band 0(800)</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="LedStopFreq">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>167</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1,000.000 MHz</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>217</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency
Step</string>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="LedCenterFreq">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>67</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1,000.000 MHz</string>
    </property>
    <property name="maxLength">
     <number>13124124</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>532</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel
Step</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>325</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Span</string>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="LedStartFreq">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>117</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1,000.000 MHz</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="LedChannel">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>482</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="LedSpan">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>317</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1,000.000 MHz</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>267</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency
Offset</string>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="LedOffsetFreq">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>267</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0.000 MHz</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_FullSpan">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>367</y>
      <width>216</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(255,255,255,1);
	padding:0px;
	text-align:center;
}
QPushButton
{
	
	background-image: url(:/rtsa/image/btn_w70_normal.png);
}
QPushButton:pressed
{
	
	background-image: url(:/rtsa/image/btn_w70_click.png);
}*/
#RtSa_Dialog_FreqSetup[status = &quot;indoor&quot;]
#pb_FullSpan
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: rgba(255, 255, 255, 1);	
	background-image: url(:/rtsa/image/btn_w216_nor.png);
}
#RtSa_Dialog_FreqSetup[status = &quot;indoor&quot;]
#pb_FullSpan:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}

#RtSa_Dialog_FreqSetup[status = &quot;outdoor&quot;]
#pb_FullSpan
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: rgba(0, 0, 0, 1);	
	background-image: url(:/rtsa/image/outdoor/btn_w216_nor.png);
}
#RtSa_Dialog_FreqSetup[status = &quot;outdoor&quot;]
#pb_FullSpan:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}

#RtSa_Dialog_FreqSetup[status = &quot;night&quot;]
#pb_FullSpan
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: #FBDBA7;
	background-image: url(:/rtsa/image/btn_w216_nor.png);
}
#RtSa_Dialog_FreqSetup[status = &quot;night&quot;]
#pb_FullSpan:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}</string>
    </property>
    <property name="text">
     <string>Full Span</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="LedChStep">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>532</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>432</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel
Standard</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>167</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop
Frequency</string>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="LedFreqStep">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>217</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1.000 MHz</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>582</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel
Type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Uplink">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>582</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Uplink</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">ChannelType</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pb_Downlink">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>582</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Downlink</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">ChannelType</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>19</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cancel">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_FreqSetup[status = &quot;indoor&quot;]
#pb_cancel
{
background-color:rgba(0,0,0,0);
background-image: url(:/rtsa/image/btn_close_normal.png);
}
#RtSa_Dialog_FreqSetup[status = &quot;indoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/rtsa/image/btn_close_click.png);
}

#RtSa_Dialog_FreqSetup[status = &quot;outdoor&quot;]
#pb_cancel
{
background-color:rgba(0,0,0,0);
background-image: url(:/rtsa/image/btn_close_normal.png);
}
#RtSa_Dialog_FreqSetup[status = &quot;outdoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/rtsa/image/btn_close_click.png);
}
#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;]  
#pb_cancel
{
background-color:rgba(0,0,0,0);
border-image: url(:/rtsa/image/night/btn_close_normal.png);
}
#RtSa_Dialog_FreqSetup[status  =  &quot;night&quot;]  
#pb_cancel:pressed
{
border-image: url(:/rtsa/image/night/btn_close_click.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>46</y>
      <width>212</width>
      <height>2</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RtSa_LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">rtsa_lineedit.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ChannelType"/>
 </buttongroups>
</ui>
